src/css/components/_progress.scss

```
@@ -80,6 +80,23 @@
   }
 }
 
+// Remove content from play-progress when using SVGs.
+.vjs-svg-icons-enabled .vjs-play-progress {
+  &:before {
+    content: none !important;
+  }
+}
+
+.vjs-play-progress .vjs-svg-icon {
+  position: absolute;
+  top: -0.3em;
```

```suggestion
  top: -0.3.5em;
```
This looks better to me on Chinese and non-Chinese Chrome